src/core/api/media_element_track_choice_manager.ts

```diff
@@ -902,6 +903,23 @@ export default class MediaElementTrackChoiceManager
       };
     }
   }
+
+  /**
+   * Enable an audio track (and disable all others), based on its index in the
+   * `this._audioTracks` array.
+   * @param {number} index}
+   */
+  private _enableAudioTrackFromIndex(index : number) : void {
+    assert(index < this._audioTracks.length);
+    for (let i = 0; i < this._audioTracks.length; i++) {
+      if (i !== index) {
```

Comment:
Because we're handling browser APIs which are known to have multiple compatibility issues, I took what I though was the safest way: first disable all other audio tracks, then enable the one we want.

Also the code you wrote wouldn't work because we would be cloning the `audioTrack` object. Here `audioTrack` is a HTMLMediaElement's [AudioTrack](https://developer.mozilla.org/en-US/docs/Web/API/AudioTrack) which we want to mutate to be able to disable / enabling it.
Cloning the object (which is what happens when creating another object destructuring the audiotrack's properties) is just going to create an object copying audioTrack's property but without any link to the HTMLMediaElement for the browser.